mkdocs.yml

```diff
@@ -28,6 +28,8 @@ nav:
   - Test suite:
     - Overview: test-suite/index.md
     - Installation & configuration: test-suite/installation-configuration.md
+    - ReFrame configuration file: test-suite/ReFrame-configuration-file.md
+    - Available tests: test-suite/available-tests.md
```

Comment:
Maybe switch available tests page with usage page? Seems more logical to me when going through the docs.

Reply:
Makes sense to me. I will change it.